pkgs/development/python-modules/pyelftools/default.nix

```
@@ -1,31 +1,34 @@
 { stdenv
 , buildPythonPackage
-, fetchPypi
+, fetchFromGitHub
 , python
 }:
 
 buildPythonPackage rec {
   pname = "pyelftools";
-  version = "0.26";
+  version = "unstable-2020-09-23";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "86ac6cee19f6c945e8dedf78c6ee74f1112bd14da5a658d8c9d4103aed5756a2";
+  src = fetchFromGitHub {
+    owner = "eliben";
+    repo = pname;
+    rev = "ab84e68837113b2d700ad379d94c1dd4a73125ea";
+    sha256 = "sha256-O7l1kj0k8bOSOtZJVzS674oVnM+X3oP00Ybs0qjb64Q=";
   };
 
   checkPhase = ''
```

to fix it
```suggestion
  doCheck = stdenv.is64bit;
  checkPhase = ''
```

and please revert my previous suggestion